actions/telemetry_actions.jsx

```diff
@@ -14,6 +17,15 @@ const SUPPORTS_MEASURE_METHODS = isSupported([
     performance.clearMeasures,
 ]);
 
+export function isTelemetryEnabled(state = store.getState()) {
```

Comment:
Maybe remove the default value from here? If we're only calling `shouldTrackPerformance` without an argument, I think it'd be best to have that be the only place its done to keep people using the other selectors correctly.